config/credentials.yml.enc

```diff
@@ -1 +1 @@
-QEIp0GrD3RqtB8tObvnL8pmQqQ+W2rbW+hH7h94hgLAuj8LwXwUKC0r2LgcMlnulERaG7ZlUu8YteDKn0jTiwmvinZGEHP0Zz1hrqlDwUxGldkgk/HDpbqJmP9NnFcnkrQZILEEM62ESGsEjqM1+8ChF3Sy1rcm6btyUyCeQEJnTGDIyh769ZPDjZEgdPlmmZexqborYqOwviOvldp8LoKwr76C1ADeL5KWLnRwx7pzUnxIh2zZlctLPSO7smG6OpbZdSszP6eiLna3XwfBJB1Rm/uZekf+SyPpDQoNh2/JCg2tax/IjPr55pB6WHL6Q30pr9VQiKHnmXsYRUx5rpUyS6VUNVAbsNE911o2u5S9egNgFOkE4ZBwdRr7zudM32neB1gSXH+VPA34GqEK62v+1wsUhEZ8ksook--XClyVe5X/M3wlKcp--KMVoZGB/chicHLSO/glEDw==
\ No newline at end of file
+6GlVR4usER2+23YGl33z+gAxF+Dr4MoPJd5/QefjOQBH1tBGiSlSaiX53XhgzFI/f3WiU/plfs3LspjbDa+/Yx7AYUinHxzcrBbCIHpB+iHxNv/LcszjNZENfdGg1CAj9gVQjddDdjBfQn3ezRq3dKeIUaR88RYIClHZIyPjiPqPU7edeqDH+TnC/pEbA39nY0vHN4EWcA==--SbFsugjEhuEySpFq--XYiS7wn8FGl/dt3zleBnNg==
```

Comment:
I'm not clear on why this needs to be replaced?

Reply:
Otherwise I'm getting: `ActiveSupport::MessageEncryptor::InvalidMessage` or `Rails.application.credentials.secret_key_base` is just `nil` (e.g. it can't decrypt the credentials with the current key in the repo)

Reply:
Rebase to master and this problem should go away.